configure.ac

```diff
@@ -53,6 +53,16 @@ AC_DEFUN([WITH_SAVE_ENV],
 [RESTORE_ENV]
 )dnl
 
+dnl AC_DEFINE*() wrappers.
+AC_DEFUN([PY_DEFINE], [dnl
+  AC_DEFINE([[Py_]$1], [$2], [$3])dnl
```

Comment:
Out of curiosity, do you want to allow defining private macros (namely `_Py`) or not?